main.go

```diff
@@ -22,7 +22,7 @@ func SimulateLootRNG() {
 	rand.Seed(time.Now().UnixNano())
 
 	nCPU := runtime.NumCPU()
-	rngTests := make([]chan []int, nCPU)
+	rngTests := make([]chan []int, 0, nCPU)
```

Comment:
cmiiw ya btw ini kalo ditaroh 0, entar pas looping dia malah ga looping karena lengthnya 0

Reply:
Oh apa iya yaa, berarti for loopnya harus diubah yaa.. wait wait dicoba dulu wkwk
Thank you buat inputnyaaa

Reply:
Oh iya kak, tadi tuh karena ga ke loop ternyata wkwk Thank you ya kak sudah di koreksi 🙏